src/shared/test/is-object-test.js

```diff
@@ -0,0 +1,22 @@
+import { isObject } from '../is-object';
+
+describe('is-object', () => {
```

Comment:
The top-level describe block is not needed here, although some older tests might have them unnecessarily. We allow multiple top-level blocks in a test file, although very few happen to use that. For modules with a single export, there is normally one top-level block matching the export name.